docs/recipes/VaryNumAnimals.md

```diff
@@ -0,0 +1,47 @@
+# Varying the number of animals in multi-animal DeepLabCut
+
+In case you have videos of the same experimental setting where the number of animals is changing between videos, DeepLabCut can deal with that like a champ. 
+
+## In case of a new project
+
+If you are just now setting up the project, then the most straightforward way to go about this is setting the max number of animals in the config file.
+
+e.g. If there are 2 and up to 6 unicorns per video, then it's best to put them all in the config file. If you can identify them then you can set `identity: true` and:
+```yaml
+individuals:
+  - jonathan
+  - samuel
+  - miranda
+  - lucius
+  - vera
+  - muriel
+```
+
+If you cannot tell the unicorns apart:
+```yaml
+individuals:
+  - unicorn 1
+  - unicorn 2
+  - unicorn 3
+  - unicorn 4
+  - unicorn 5
+  - unicorn 6
+```
+
+When labeling, it is always best to label hard frames, i.e. that have all the 6 animals. Labeling videos with less animals and even 1 works in principle, but practically assembly and tracking performance will be better when training with "hard frames" that have many occlusions and interactions. This is a general recommendation with DeepLabCut, try to label frames that reflect the statistics of luminance, movement & interactions. 
```

Comment:
i..e --> i.e., 

what are hard frames? Frames with animals never touching but 6 animals are not "hard," please use concrete language.